roles/nuage_node/tasks/iptables.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: IPtables | Get iptables rules
+  command: iptables -L --wait
+  register: iptablesrules
+  always_run: yes
+
+- name: Allow traffic from overlay to underlay
+  command: /sbin/iptables -I FORWARD 1 -s {{ hostvars[groups.oo_first_master.0].openshift.master.sdn_cluster_network_cidr }} -j ACCEPT -m comment --comment "nuage-overlay-underlay" 
```

Comment:
@vishpat I'm good with adding this with the addition of `--wait` to the iptables command here.